src/components/Integrations/BigQuery/FreeAccount/FreeAccount.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { BigQueryLogo } from './styles/styles';
+import { useIntl } from 'react-intl';
+import datastudiogif from '../../../../img/google-data-studio.gif';
+import { FormattedMessageMarkdown } from '../../../../i18n/FormattedMessageMarkdown';
+
+interface IFreeAccountProps {}
+
+const FreeAccount: React.FunctionComponent<IFreeAccountProps> = (props) => {
```

Comment:
We try to use named exports like this:
```suggestion
export const FreeAccount: React.FunctionComponent<IFreeAccountProps> = (props) => {
```

Reply:
suggest accepted and applied